From 59287a0bd0e4d58a4d2b434bde3db9b2376ac7c1 Mon Sep 17 00:00:00 2001
From: Angel Pons <th3fanbus@gmail.com>
Date: Tue, 16 May 2023 11:13:17 +0200
Subject: [PATCH] chipset_enable.c: Drop `_LARGEFILE64_SOURCE`

This file does not access any large files, so there's no need to define
this feature test macro.

Change-Id: I866cfa2f996eeea5846e5d9189647ad7a4a4e3e4
Signed-off-by: Angel Pons <th3fanbus@gmail.com>
Reviewed-on: https://review.coreboot.org/c/flashrom/+/75271
Tested-by: build bot (Jenkins) <no-reply@coreboot.org>
Reviewed-by: Nico Huber <nico.h@gmx.de>
Reviewed-by: Anastasia Klimchuk <aklm@chromium.org>
Reviewed-by: Alexander Goncharov <chat@joursoir.net>

Upstream: https://github.com/flashrom/flashrom/commit/59287a0bd0e4d58a4d2b434bde3db9b2376ac7c1
Signed-off-by: James Hilliard <james.hilliard1@gmail.com>
---
 chipset_enable.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/chipset_enable.c b/chipset_enable.c
index 9d3df5f8..3846ecfb 100644
--- a/chipset_enable.c
+++ b/chipset_enable.c
@@ -24,8 +24,6 @@
  * Contains the chipset specific flash enables.
  */
 
-#define _LARGEFILE64_SOURCE
-
 #include <stdbool.h>
 #include <stdlib.h>
 #include <string.h>
-- 
2.34.1

